src/options.toml

```diff
@@ -34,6 +34,43 @@
 #All of these options are required
 #If they are not provided the program will exit early
 [BCs]
+    # Tells the program that we'll have changing BCs
+    # If left out of the file this defaults to a value of false
+    changing_ess_bcs = false
+    # The below gives an example of how to make use of changing essential BCs
+    # where we have the sample cyclically loaded.
```

Comment:
Cyclic loading might be easier to specify if you add a feature where the specified velocity can be a function of time (say through a 1D table), rather than making users spell things out in this rather verbose way. 

That way of doing it might dovetail nicely with a capability for constant velocity gradient type BCs.

Reply:
I believe I should be able to add a cyclic or uniaxial type option. Since, we know the applied boundary faces remain constant through out time. However, it should be up to the user to specify the step number that things are changing and what the new velocity should be. I say this because cyclic examples can get pretty interesting where you might change the strain things are being cycled at or potentially the strain rates for which it is occurring. Alternatively, they might want to unload slightly at a given point in-order to mimic an experiment such as a HEXD cyclic exp where it is common to see the sample unloaded slightly when imagining is being taken, so you don't potentially have the sample creep.

I can add this feature in a new PR that's smaller in scope.